src/datastore/postgres-store.ts

```diff
@@ -306,6 +306,10 @@ interface TransferQueryResult {
   amount: string;
 }
 
+export interface RawTxQueryResult {
+  raw_tx: string;
```

Comment:
This is stored as a `bytea` column type, which means this return value should be `raw_tx: Buffer`?